bmcremedy_connector.py

```diff
@@ -61,9 +69,17 @@ def _decrypt_state(self, state, salt):
             return state
 
         token = state.get("token")
+        oauth_token = state.get("oauth_token")
         if token:
             state["token"] = encryption_helper.decrypt(token, salt)
-
+        if oauth_token:
+            token_list = ['access_token', 'id_token', 'refresh_token']
```

Comment:
I see that the same list is used in multiple places. Maybe we can move it to consts and import it from there?

Reply:
Yes, I will move it to const and import from there.